specs/engage-digital_openapi3.yaml

```diff
@@ -2260,15 +2260,13 @@ paths:
             type: string
         - description: >-
             To filter interventions on given identity_id(s). Can be a single
-            value or an array.
+            value or an array of string.
           explode: true
           in: query
           name: identity_id
           required: false
           schema:
-            items:
-              type: string
-            type: array
+            type: string

```

Comment:
I changed this one to be only string because this is its "default" value, and there's no `s` in `identity_id`. I also updated the description to detail it's an array of string.

Reply:
Ok fine for me, no need to add brackets then